core/i18n/resource/i18n/pt-PT.csv

```diff
@@ -0,0 +1,43 @@
+"This feature is not implemented yet! Please take a look at https://github.com/DivanteLtd/vue-storefront/issues for our Roadmap!","Esta funcionalidade ainda não foi implementada! Por favor consulte o nosso Rodmap em https://github.com/DivanteLtd/vue-storefront/issues!"
+"The product is out of stock and cannot be added to the cart!","O produto não se encontra em stock e não pode ser adicionado ao Cesto de Compras!"
+"Product has been added to the cart!","O produto foi adicionado ao Cesto de Compras!"
+"Internal validation error. Please check if all required fields are filled in. Please contact us on contributors@vuestorefront.io","Erro interno de validação. Por favor verifique se todos os campos obrigatórios estão preenchidos. Por favor contate-nos em contributors@vuestorefront.io"
+"Product {productName} has been added to the compare!","Produto {productName} foi adicionado a Comparar Produtos!"
+"Product {productName} has been removed from compare!","Produto {productName} foi removido de Comparar Produtos!"
+"Product {productName} has been added to wishlist!","Produto {productName} foi adicionado à Lista de Desejos!"
+"Product {productName} has been removed from wishlit!","Produto {productName} foi removido da Lista de Desejos!"
+"Account data has successfully been updated","Os dados da conta foram atualizados"
+"Newsletter preferences have successfully been updated","As suas preferências para a Newsletter foram atualizadas."
+"Reset password feature does not work while offline!","A funcionalidade de Recuperação de Senha não funciona sem ligação à internet!"
+"You are logged in!","Você está com sessão inicializada!"
+"Please fix the validation errors","Por favor corrija os erros de validação"
+"Product price is unknown, product cannot be added to the cart!","O preço do produto é desconhecido, o produto não pode ser adicionado ao Cesto de Compras!"
+"My Account","A Minha Conta"
+"Type what you are looking for...","Escreva o que está procurando..."
+"Home Page","Página Inicial"
+"Checkout","Finalizar Compra"
+"Subtotal incl. tax","Subtotal c/ imposto"
+"Grand total","Total"
+"Field is required","Campo obrigatório"
+"Field is required.","Campo obrigatório."
+"You're logged out","Você não está com sessão inicializada"
+"Compare Products","Comparar Produtos"
+"404 Page Not Found","404 Página Não Encontrada"
+"Error with response - bad content-type!","Erro na resposta do servidor - bad content-type!"
+"Unhandled error, wrong response format!","Erro desconhecido, formato de resposta errado!"
+"not authorized","não autorizado"
+"Internal Application error while refreshing the tokens. Please clear the storage and refresh page.","Erro Interno da Aplicação ao atualizar as tokens. Por favor limpe o cache do seu browser e recarregue a página."
+"Proceed to checkout","Ir para Finalizar Compra"
```

Comment:
Talvez apenas `Finalizar compra`

Reply:
Normalmente, nos sites portugueses Checkout é traduzido por 'Finalizar Compra' e como tal "Proceed to checkout" passa a "Ir para Finalizar Compra"